src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -39,14 +39,6 @@
 
 _CompleteFunct = CFUNCTYPE(None, c_void_p, c_void_p)
```

Comment:
Lets remove unused imports as well as part of this PR (time and Thread)

Also global variables (_CompleteFunct and _ErrorFunct according tothe code scanner)